pkg/ucp/dataprovider/options.go

```diff
@@ -37,6 +37,8 @@ type StorageProviderOptions struct {
 
 	// InMemory configures options for the in-memory store. Will be ignored if another store is configured.
 	InMemory InMemoryOptions `yaml:"inmemory,omitempty"`
+
+	PostgreSQL PostgreSQLOptions `yaml:"postgresql,omitempty"`
```

Comment:
pls add comment~: // PostgreSQLOptions configures options for connecting to a PostgreSQL database. Will be ignored if another store is configured.

Reply:
Adding